content/pages/about.md

```diff
@@ -7,11 +7,11 @@ Chennaipy is a Python user-group, where Pythonistas in and around
 Chennai come together to share their knowledge and to contribute to
 the promotion of Python.
 
-<!-- NOTE: Upcoming event template, update and uncomment 
+ <!-- <div class="message">NOTE: Upcoming event template, update and uncomment</div> -->
 <div class="cp-info-box">
-  <b>Upcoming Events:</b> <a href="/meetup-apr-2015.html">April Meetup</a>
+  <b>Upcoming Events:</b> <a href="http://www.meetup.com/Chennaipy/events/222732435/" target="_blank">June Meetup</a>
```

Comment:
The June meetup still isn't announced. Lets keep this for when we announce it on the mailing list and the meetup.